src/Disks/IO/ReadBufferFromAzureBlobStorage.cpp

```diff
@@ -262,6 +259,51 @@ size_t ReadBufferFromAzureBlobStorage::getFileSize()
     return *file_size;
 }
 
+size_t ReadBufferFromAzureBlobStorage::readBigAt(char * to, size_t n, size_t range_begin, const std::function<bool(size_t)> & progress_callback) const
```

Comment:
`test_storage_azure_blob_storage/test.py::test_schema_inference_cache ` test uses parallel read, 
any other suggestion for tests?

Reply:
> test_storage_azure_blob_storage/test.py::test_schema_inference_cache  test uses parallel read

not obvious that it does :) because there are no explicit checks for it, better to explicitly check that it is being used

Reply:
added similar test with a larger size & a log check